design/one-pager-stack-relationship-labels.md

```diff
@@ -0,0 +1,49 @@
+# Stack Parent/Child Relationship Label Spec
+
+- Owner: Steven Rathbauer (@rathpc)
+- Reviewers: Crossplane Maintainers
+- Status: Draft
+
+## Proposal
+
+[crossplaneio/crossplane#934](https://github.com/crossplaneio/crossplane/issues/934)
+
+## Problem
+
+There is currently no easy way to get a list of all resources that are owned by a parent resource. We can leverage CRD labels to make this easier. As long as we have a standard format for doing so.
```

Comment:
Would you be able to wrap long lines at a shorter length? It makes them easier to comment on in GitHub, since each line is shorter : )